### PR TITLE
Implements: the design changes for the Pages row 

### LINKED ISSUE #19348: UI Modernization Pages: Update Page list layouts
Figma Design eYeHXEMDbnFptE40xUqZ2T-fi-836_5909
```[tasklist]
### Tasks
- [x] Add new page list layout
- [x] Add new page list skeleton layout
- [x] Remove old layout files
- [x] Remove old string resources
- [x] Apply approriate styles according to Figma designs
- [x] Update adapter and view holders
```

### PR DESCRIPTION
**Pull request by AjeshRPai**
## Closes
#19348

## Description
This PR 
#### 1. Updates the pages row design 

- Changes the font of the Home page template view
- Changes the font of page title 
- Changes the size of featured image to 48dp 
- Changes the font of page author, date and type of page **text**
- Changes the order of date, author and type of page text to type of page, date and author

|  Before |  After  | 
|---|---|
|  Screenshot 2023-11-02 at 9 52 17 AM|  Screenshot 2023-11-01 at 12 17 29 PM |   
|  Screenshot 2023-11-02 at 10 17 43 AM | Screenshot 2023-11-02 at 10 37 39 AM |
| Screenshot 2023-11-02 at 9 52 33 AM | Screenshot 2023-11-01 at 6 42 21 PM |

#### 2. Adds the logic to show/hide the author name depending on -
1.  Site is not  a Singe user site
2.  Site has edit pages capabilities
3.  Site is accessed through WP com
4.  Whether the author filter selection is ME 

## To test:

**Design Review** 

1. Login to the Jetpack app with a site having pages
2. Go to Pages
3. Verify that the pages list is shown as per design

In order to verify the set posts as page icon in Pages, Create a site using Personal → I am a photographer theme. That was one of the themes supporting posts as page. 

**Regression testing - All pages are shown**

1. Login to the Jetpack app with a site having pages
2. Go to Pages
3. Verify that all pages are shown

**Author name is not shown when its a single user site**

1. Login to the Jetpack app with a single user site
2. Go to Pages
3. Verify that all pages are shown
8. Verify that author name is not shown

**Regression - Author name is not shown when the author Filter is ME**

1. Login to the Jetpack app with a site having multiple users
2. Go to Pages
3. Verify that all pages are shown
4. Select ME from the author filter
9. Verify that the author name is not shown

**Author filter is shown for Atomic Sites**

1. Login to the Jetpack app with an Atomic Site with Multiple users
2. Go to Pages 
3. Verify that all pages are shown 
4. Verify that Me/Everyone filter is shown on the toolbar 
5. Verify that the filter works as expected 

## Regression Notes

1. Potential unintended areas of impact
    
    Pages list is not shown as intended 
    
2. What I did to test those areas of impact (or what existing automated tests I relied on)
    
    Manual testing 
    
3. What automated tests I added (or what prevented me from doing so)
    
    Manual testing 
    

PR submission checklist:

- [x]  I have completed the Regression Notes.
- [x]  I have considered adding accessibility improvements for my changes.
- [x]  I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

UI Changes testing checklist:

- [x]  Portrait and landscape orientations.
- [x]  Light and dark modes.
- [x]  Fonts: Larger, smaller and bold text.
- [x]  High contrast.
- [x]  Talkback.
- [x]  Languages with large words or with letters/accents not frequently used in English.
- [x]  Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [x]  Large and small screen sizes. (Tablet and smaller phones)
- [x]  Multi-tasking: Split screen and Pop-up view. (Android 10 or higher)

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt [hunk 1]
```diff
@@ -94,6 +95,11 @@ class PageListViewModel @Inject constructor(
     private val PageModel.isPostsPage: Boolean
         get() = remoteId == pagesViewModel.site.pageForPosts
 
+    private val showAuthorName: Boolean by lazy {
+        // show if the site is a single user site and the users in the site has the capability to edit/add pages
+        !pagesViewModel.site.isSingleUserSite && pagesViewModel.site.hasCapabilityEditOthersPages
```
Comment (zwarm) [inline WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt:100]: ❓ Should this also include the `site.isUsingWpComRestApi` check?

File: WordPress/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -507,7 +507,7 @@
     <dimen name="two_line_list_item_height">72dp</dimen>
     <dimen name="page_list_divider_height">40dp</dimen>
     <dimen name="page_list_row_min_height">80dp</dimen>
-    <dimen name="page_list_featured_image_size">40dp</dimen>
+    <dimen name="page_list_featured_image_size">48dp</dimen>
```
Comment (zwarm) [inline WordPress/src/main/res/values/dimens.xml:510]: 48dp looks kind of smallish, but correct. For reference, the post list is using 64dp as the featured image size. FYI @osullivanchris

**Review by zwarm — COMMENTED** [review]
@AjeshRPai - So far, so good, but I have some questions before approving. 

The first two are inlined, the next one has to do with the author filter. When "everyone" is selected I expect to see an author name for all pages, even if I am the author. See the screenshot below.

Alt desc

File: WordPress/src/main/res/values/dimens.xml (hunk 2 above)
Comment (AjeshRPai) [reply WordPress/src/main/res/values/dimens.xml:510]: Yup I confirmed this with @osullivanchris 👍🏼

File: WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt (hunk 1 above)
Comment (AjeshRPai) [reply WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt:100]: I don't think thats needed. Even for a self hosted site, the author name was being shown earlier and I don't think we should change that. I tested the changes in this PR and the trunk using a Jurassic site with multiple users. In both builds, I can see the name of the author

**Comment by AjeshRPai** [comment]
Hey @zwarm 
>The first two are inlined, the next one has to do with the author filter. When "everyone" is selected I expect to see an author name for all pages, even if I am the author. See the screenshot below.

Can you tell me what kind of a site u are using here 🤔 . It should show author if the filter is shown and the selected filter is everyone.

File: WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt (hunk 1 above)
Comment (zwarm) [reply WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt:100]: Gotcha, I thought this was wrapping whether the filter should be shown. 🙇

**Comment by zwarm** [comment]
@AjeshRPai 

> Can you tell me what kind of a site u are using here 🤔 . It should show author if the filter is shown and the selected filter is everyone.

Atomic site with multiple users. And I'm not signed in as the owner, but as en editor

**Comment by AjeshRPai** [comment]
Hey @zwarm 

I couldn't replicate this issue. As we discussed, I am guessing this page is a shopping cart or something like that. In the pages list, all the pages are treated as posts, and then the author field is looked at in the model. I think there is a missing logic when the page is a cart. 

Since this is a corner case scenario and the issue was not introduced due to the changes in this PR, I say let's merge this PR, find the root cause, and fix it in a separate PR. 

WDYT?

**Comment by zwarm** [comment]
> WDYT?

I agree, let's keep it as is. Thanks for looking into it. 👍

**Comment by zwarm** [comment]
FYI  @osullivanchris - I am going to merge this PR into our feature branch so we push everything to trunk. Any design issues will be taken up in a new PR.

**Pull request merged**